Woof/Woof/View/DetailedPetSitterView/DetailedPetSitterView.swift

```diff
@@ -0,0 +1,34 @@
+import SwiftUI
+
+/// A view displaying detailed information about the sitter.
+struct DetailPetSitterView: View {
+    /// The view model responsible for providing data to the view.
+    @ObservedObject var viewModel: DetailSitterViewModel
+
+    var body: some View {
+        VStack {
+            AvatarView(url: viewModel.imageURL)
+            FiveStarRatingView(rating: viewModel.rating)
+            TextWithIconLabelView(
+                iconName: Image.IconName.filledPerson,
+                text: viewModel.fullName
+            )
+            TextWithIconLabelView(
+                iconName: Image.IconName.phone,
+                text: viewModel.phoneNumber
+            )
+            Text(viewModel.bio)
+                .padding()
+            Spacer()
+        }
```

Comment:
Check the padding please carefully.
```suggestion
      VStack {
            AvatarView(url: viewModel.imageURL)
            FiveStarRatingView(rating: viewModel.rating)
            TextWithIconLabelView(
                iconName: Image.IconName.filledPerson,
                text: viewModel.fullName
            )
            TextWithIconLabelView(
                iconName: Image.IconName.phone,
                text: viewModel.phoneNumber
            )
            Text(viewModel.bio)
            Spacer()
        }
        .padding()
```

Reply:
Thanks, fixed it.

Reply:
The screenshot  in the description is updated.

Reply:
Sorry for the previous unclear comment! I Fixed paddings according to the original screenshot.